firebase-installations/src/main/java/com/google/firebase/installations/local/FirebaseInstallationIdCache.java

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.local;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.firebase.FirebaseApp;
+
+/**
+ * A layer that locally caches a few Firebase Installation attributes on top the Firebase
+ * Installation backend API.
+ */
+public class FirebaseInstallationIdCache {
```

Comment:
As this class is not used by developers, we can use a more brief name, for example FiidCache, same for the cache entry.

Not a comment, on your call.

Reply:
Done. Thanks.